platform/persistence/platform-persistence-core-listeners/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,30 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD description="System Alert Listener"
+         name="Alert Listener" id="org.codice.ddf.persistence.events.AlertListener">
+
+        <AD description="The periodic interval on which undismissed alert events should be resent."
+                name="Periodic Alert Interval (minutes)" id="period" required="true"
+                type="Integer" default="1440">
```

Comment:
Don't think it can be avoided here unless I change this to be hours which would be a little easier to read but give less granular control. Not sure what's best here. Probably don't need anything more granular than hours. What do you think?